tests/pytorch/testproblems/test_cifar10_resnet32nobn.py

```diff
@@ -0,0 +1,128 @@
+"""Test the CIFAR-10 problem with the ResNet32 (BN replaced by Identity).
+
+Check that BackPACK's extensions and ViViT works with this architecture. 
+Also check compatibility with DeepOBS by running the StandardRunner.
+"""
+
+from copy import deepcopy
+
+import torch
+from backpack import backpack, extend
+from backpack.core.derivatives.convnd import weight_jac_t_save_memory
+from backpack.extensions import BatchGrad, DiagGGNExact
+from deepobs import pytorch as pt
+from deepobs.pytorch.testproblems.cifar10_resnet32nobn import cifar10_resnet32nobn
+from torch.optim import SGD
+from vivit.linalg.eigh import EighComputation
+
+
+# Set up testproblem
+testproblem = cifar10_resnet32nobn(batch_size=16)
+torch.manual_seed(0)
+testproblem.set_up()
+
+# Extract model
+model = testproblem.net.eval()
```

Comment:
Not necessary because no BN

Reply:
True. I will leave it there until we have decided whether we use BN or not.

Reply:
Since we do use BN layers now, I left it like this.